spec.md

```diff
@@ -180,6 +180,19 @@ contributors is below:
 - [`@tdb-alcorn`](https://github.com/tdb-alcorn)
 - [`@youngar`](https://github.com/youngar)
 
+# File Preamble
+
+A firrtl file begins with a magic string and version identifier indicating the 
+version of this standard the file conforms to 
+(see [@sec:versioning-scheme-of-this-document]).  This will not be present on 
+files generated according to versions of this standard prior to the first 
+versioned release of this standard to include this preamble.
+
+``` firrtl
+firrtl version 1.1.0
```

Comment:
```suggestion
FIRRTL version 1.1.0
```

i think your language table below has this as all-caps?

Reply:
yea, meant to check that.  thanks.